### PR TITLE
[ZIP-12] Standardisation of ZIP processes

### PR DESCRIPTION
**Pull request by evesnow91**
Standardisation of ZIP processes including forum discussion, signalling, and snapshot voting.

The order will look like this:
> propose > discuss > codify > vote > conclude > execute

### CODE REVIEW

**Review by renlulu — APPROVED** [review]
Thanks hanwen, looking forward to the governance!

File: zips/zip-12.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+#### Formal Voting Phase
+
+Snapshot is used for formal, binding votes. The user who authored the ZIP will also create the proposal on Snapshot[3]. Snapshot requires > 0.1% of the total supply of gZIL[[4](https://viewblock.io/zilliqa/address/zil14pzuzq6v6pmmmrfjhczywguu0e97djepxt8g3e)] in a user’s wallet to create a proposal. If the author does not meet this requirement then contact a moderator who will submit the proposal on your behalf. Before creating the Snapshot vote, please wait for a moderator to assign your ZIP a number and begin your Snapshot title with it.
+
+A minimum of 96 hours (5 days) for voting, and a 20% quorum requirement is required for ZIPs voting processes. This wide time period for voting, coupled with an active communication of ZIPs via the governance forum and on social media channels should help ensure that no ZIPs will slip through and no malicious ones are approved.
```
Comment (bb111189) [inline zips/zip-12.md:44]: 5 days is 120 hours

File: zips/zip-12.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+### Other Uses for Snapshot
+
+Snapshot may still be used for informal signal voting, community contests and grantsDAO disbursement, but its primary purpose will be to conduct formal, binding votes for ZIPs.
+
+# TL;DR
+
+1. All proposals must be discussed for at least 3 days with a forum poll before they will be assigned a ZIP number by a ZIP editor and moved to Snapshot voting.
+2. The Snapshot vote will be binding, and must be open for at least 5 days. At least 20% quorum (> 20% of token total supply participating in the voting process) must be reached, and at least 50% must vote “For” the proposal for it to pass.
+3. Most open-source protocol updates do not require ZIPs; this process only governs those changes or proposals that wish to be discussed and enacted as ZIPs.
```
Comment (bb111189) [inline zips/zip-12.md:62]: Most open-source protocol updates do not require ZIPs

Certain core update may be written in the form of ZIP. I think for such a proposal, it no need to undergo through ZIP

**Review by bb111189 — APPROVED** [review]

**Review by ansnunez — APPROVED** [review]

**Comment by evesnow91** [comment]
Please merge this now that this ZIP is officially formalised via the Snapshot vote:
https://governance.zilliqa.com/#/gzil/proposal/QmVRudHVZEk95odgKot8nHW42NRWnEPCdExt21fxtxxMCB

**Pull request merged**